GVFS/GVFS/CommandLine/IncludeVerb.cs

```diff
@@ -0,0 +1,173 @@
+﻿using CommandLine;
+using GVFS.Common;
+using GVFS.Common.Database;
+using GVFS.Common.FileSystem;
+using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace GVFS.CommandLine
+{
+    [Verb(IncludeVerb.IncludeVerbName, HelpText = "Clone a git repo and mount it as a GVFS virtual repo")]
+    public class IncludeVerb : GVFSVerb.ForExistingEnlistment
+    {
+        private const string IncludeVerbName = "include";
+
+        [Option(
+            'a',
+            "add",
+            Required = false,
+            Default = "",
+            HelpText = "A semicolon-delimited list of folders to include. Wildcards are not supported.")]

```

Comment:
We will want to allow `--stdin-add` or something so we don't run out of room in the command-line arguments when scoper runs. Can be delayed to a later PR.